### DISCUSSION TITLE
Comedy Central Latino | (México)

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#55] [Show and tell]
### Channel Name

Comedy Central

### Channel ID

ComedyCentralNorth.us

### Stream URL

https://cdn.prepro.cvattv.com.ar/live/vod/ComedyCentral/SA_Live_WebVTT_FTA/ComedyCentral.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Sergiothewitcher** [comment]
Just a comment: I would not recommend adding any stream that is under the domain "cvattv.com.ar". Those are legitimate streams from Flow, a paid service... and when they see their stream being illegally shared, they block it...

**Comment by freearhey** [comment]
https://cdn.prepro.cvattv.com.ar/live/vod/ComedyCentral/SA_Live_WebVTT_FTA/ComedyCentral.m3u8
<img width="509" alt="image" src="https://user-images.githubusercontent.com/7253922/185198330-e903da76-3c6d-46eb-859e-497636c590ba.png">
via VPN
